src/usr/local/www/diag_ndp.php

```diff
@@ -128,10 +135,11 @@ function _getHostName($mac, $ip) {
 	<table class="table table-striped table-condensed table-hover sortable-theme-bootstrap" data-sortable>
 		<thead>
 			<tr>
-				<th><?= gettext("IPv6 address"); ?></th>
```

Comment:
Spacing of this block was inconsistent with the rest of the page (spaces after ?= & trailing '; ')